config/jobs/README.md

```diff
@@ -1,9 +1,146 @@
 # Kubernetes Prow Job Configs
 
-Here are prow job configs for the Kubernetes prow deployment.
+All prow job configs for [prow.k8s.io] live here.
 
-Please add jobs per org|repo/sig subdir.
+They are tested and validated by tests in [`config/tests`](/config/tests)
 
-The file basename will be used as a configmap key so they need to be unique across this directory.
+## Job Cookbook
 
-Questions? Contact @krzyzacy!
\ No newline at end of file
+This document attempts to be a step-by-step easy copy-pasta guide to the use
+of prow jobs for the Kubernetes project. It may fall out of date. For more 
+info, it was sourced from the following:
+
+- [ProwJob docs](/prow/jobs.md)
+- [Life of a Prow Job](/prow/life_of_a_prow_job.md)
+- [Pod Utilities](/prow/pod-utilities.md)
+- [How to Test a Prow Job](/prow/build_test_update.md#How-to-test-a-ProwJob)
+
+### Job Types
+
+Generically, there are three typs of prow jobs:
+
+- **Presubumits** run against code in PRs
+- **Postsubmits** run after merging code
+- **Periodics** run on a periodic basis
+
```

Comment:
An opportunity here to point to jobs.md, which directly parallels this and demonstrates in more detail the API options

I might specify that it's for "more detail", since I don't think that doc isn't a prerequisite to this guide

Reply:
done